src/test/java/se/sundsvall/esigning/integration/camunda/configuration/CamundaConfigurationTest.java

```diff
@@ -0,0 +1,58 @@
+package se.sundsvall.esigning.integration.camunda.configuration;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static se.sundsvall.esigning.integration.camunda.configuration.CamundaConfiguration.CLIENT_ID;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.cloud.openfeign.FeignBuilderCustomizer;
+
+import se.sundsvall.dept44.configuration.feign.FeignMultiCustomizer;
+import se.sundsvall.dept44.configuration.feign.decoder.ProblemErrorDecoder;
+
+@ExtendWith(MockitoExtension.class)
+class CamundaConfigurationTest {
+
+	@Spy
+	private FeignMultiCustomizer feignMultiCustomizerSpy;
+
+	@Mock
+	private FeignBuilderCustomizer feignBuilderCustomizerMock;
+
+	@Mock
+	private CamundaProperties propertiesMock;
+
+	@Test
+	void testFeignBuilderCustomizer() {
+		final var configuration = new CamundaConfiguration();
```

Comment:
You could use `@InjectMocks` on this, instead of creating a new 

If you don't want to have the variables as memebers in this class, let them be parameters to the test-method instead?